lib/log.js

```diff
@@ -66,17 +66,22 @@ Lug.prototype.event = function (name, data) {
   process.stdout.write(JSON.stringify(e) + '\n')
 }
 
-Lug.prototype.activityEvent = function(event, uid, request) {
+Lug.prototype.activityEvent = function(event, request, data) {
   var info = {
-    event: event,
-    uid: uid,
```

Comment:
I really wanted to add activityEvent tests. 
Otherwise, We should maybe log an error if an activity event is sent with no uid?